situation_room/.formatter.exs

```diff
@@ -0,0 +1,5 @@
+# Used by "mix format"
```

Comment:
So, all thes files in this whole PR commit should be one directory up. These files all should live at the repository root directory, not at root/situation_room/files.